apps/studio/components/interfaces/Realtime/RealtimeSettings.tsx

```diff
@@ -138,6 +165,14 @@ export const RealtimeSettings = () => {
                           />
                         </FormControl_Shadcn_>
                         <FormMessage_Shadcn_ />
+                        {!!maxConn && field.value > maxConn.maxConnections * 0.5 && (
+                          <Admonition
```

Comment:
Maybe keep this admonition, but put a small note in the ui. That helps tie the two numbers together for users? 

![screenshot-2025-03-18-at-21 03 48](https://github.com/user-attachments/assets/98b5be2c-ac7b-4ec3-8cb8-76046c5f7431)

Reply:
hmm 50% is not a hard limit though - its only a recommendation so reckon we only show it if the users actually input a large value